app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -90,10 +102,10 @@ object AppRepository {
     private lateinit var scheduleNetworkRepository: ScheduleNetworkRepository
     private lateinit var engelsystemNetworkRepository: EngelsystemNetworkRepository
     private lateinit var sharedPreferencesRepository: SharedPreferencesRepository
+    private lateinit var sessionsTransformer: SessionsTransformer
 
-    private var onSessionsChangeListener: OnSessionsChangeListener? = null
-    private var alarmsHaveChanged = false
-    private var highlightsHaveChanged = false
+    private val mutableLoadScheduleState = MutableSharedFlow<LoadScheduleState>(extraBufferCapacity = 1)
```

Comment:
You most likely want to change this to
```suggestion
    private val mutableLoadScheduleState = MutableSharedFlow<LoadScheduleState>(replay = 1, onBufferOverflow = BufferOverflow.DROP_OLDEST)
```

Reply:
Yes. But due to the way you're using the Flow you'd probably be fine with `extraBufferCapacity = 1, onBufferOverflow = BufferOverflow.DROP_OLDEST`. However, if at some point you'll switch to exposing the Flow using `asLiveData()` you'll want the replay behavior. Otherwise you won't get the current state (a second time) e.g. when changing the device orientation.

Reply:
:heavy_check_mark: Changed in 0f158745dfc45b4fa01d17e172e471a0522c91f0.